executor/builder.go

```diff
@@ -2345,7 +2347,7 @@ func (b *executorBuilder) getAdjustedSampleRate(sctx sessionctx.Context, tid int
 	// To do a workaround for this issue, we check the approxiCount from the pd side to do a comparison.
 	// If the count from the stats_meta is extremely smaller than the approximate count from the pd,
 	// we think that we meet this issue and use the approximate count to calculate the sample rate.
-	if float64(statsTbl.Count*100) < approxiCount {
+	if float64(statsTbl.Count*5) < approxiCount {
```

Comment:
Why we change the constant here?

Reply:
Before the PR, when the table is small, we may get a large `approxiCount`. After the PR, when the table is small(only in one region), we get a more accurate `approxiCount`. In another word, the change of `approxiCount` is much larger than the true value goes smaller so we turn the constant from `100` to `5`.